sdk/dotnet/sdk/Dagger.SDK/Dagger.cs

```diff
@@ -8,7 +8,7 @@ public static class Dagger
         new(() => new Query(QueryBuilder.Builder(), new GraphQLClient()));
 
     // <summary>
-    // Connect to the Dagger Engine.
+    // Get a Query instance to start building a dag.
     // </summary>
-    public static Query Connect() => Query.Value;
+    public static Query Dag() => Query.Value;
```

Comment:
This doesnt actually connect at all. this only returns a lazy evaluated client, which is ready to make a connection, which only occurs at leaf async function calls. `Connect` feels wrong. Maybe `Query` or `Dag` make more logical sense